studio/components/interfaces/Database/Schemas/SchemaGraph.tsx

```diff
@@ -0,0 +1,341 @@
+          <header className="text-[0.5rem] leading-5 font-bold px-2 text-center bg-yellow-900 text-gray-300">
+            {data.name}
+            {targetPosition && (
+              <Handle
+                type="target"
+                id={data.name}
+                position={targetPosition}
+                className={`${hiddenNodeConnector} !left-0`}
+              />
+            )}
+          </header>
+        </div>
+      ) : (
+        <div className={`w-[${NODE_WIDTH / 2}px] rounded-lg overflow-hidden`}>
+          <header className="text-[0.5rem] leading-5 font-bold px-2 text-center bg-brand-900 text-gray-300">
+            {data.name}
+          </header>
+
+          {data.columns.map((column) => (
+            <div
+              className="text-[8px] leading-5 relative flex justify-between odd:bg-scale-300 even:bg-scale-400"
+              key={column.id}
+            >
+              <span
+                className={`${
+                  column.isPrimary ? 'border-l-2 border-l-brand-900 pl-[6px] pr-2' : 'px-2'
+                } text-ellipsis overflow-hidden whitespace-nowrap`}
+              >
+                {column.name}
+              </span>
+              <span className="px-2">{column.format}</span>
+              {targetPosition && (
+                <Handle
+                  type="target"
+                  id={column.id}
+                  position={targetPosition}
+                  className={`${hiddenNodeConnector} !left-0`}
+                />
+              )}
+              {sourcePosition && (
+                <Handle
+                  type="source"
+                  id={column.id}
+                  position={sourcePosition}
+                  className={`${hiddenNodeConnector} !right-0`}
+                />
+              )}
+            </div>
+          ))}
+        </div>
+      )}
+    </>
+  )
+}
+
+const TablesGraph: FC<{ tables: PostgresTable[] }> = ({ tables }) => {
+  const reactFlowInstance = useReactFlow()
+  const nodeTypes = useMemo(
+    () => ({
+      table: TableNode,
+    }),
+    []
+  )
+
+  useEffect(() => {
+    getGraphDataFromTables(tables).then(({ nodes, edges }) => {
+      reactFlowInstance.setNodes(nodes)
+      reactFlowInstance.setEdges(edges)
+      setTimeout(() => reactFlowInstance.fitView({})) // it needs to happen during next event tick
+    })
+  }, [tables])
+
+  return (
+    <>
+      <div style={{ width: '100%', height: '100%' }}>
+        <ReactFlow
+          defaultNodes={[]}
+          defaultEdges={[]}
+          defaultEdgeOptions={{
+            type: 'smoothstep',
+            animated: true,
+            deletable: false,
+            style: {
+              stroke: '#ededed',
+            },
+          }}
+          nodeTypes={nodeTypes}
+          fitView
+          proOptions={{
+            hideAttribution: true,
+          }}
+        >
+          <Background id="1" gap={10} color="#262626" variant={BackgroundVariant.Lines} />
```

Comment:
i might be selecting the wrong line here but on light mode the grid lines against the background are standing out too much - could we use a lighter color? (and if possible using our tailwind classes for this?)

![image](https://github.com/supabase/supabase/assets/19742402/74967fbc-e062-45c2-a719-4061bade9209)

Reply:
Updated. Unfortunately, we cannot use tailwind here.

Reply:
we could set up some tailwind custom classes to deal with some of this if we need to.
but probably no need if we just import a css file onto the NextJs page.